chassis/core/src/main/java/com/griddynamics/jagger/util/generators/WorkloadGenerator.java

```diff
@@ -19,6 +27,20 @@ static WorkloadClockConfiguration generateLoad(JLoadProfile jLoadProfile) {
             ((RpsClockConfiguration) clockConfiguration).setValue(((JLoadProfileRps) jLoadProfile).getRequestsPerSecond());
             ((RpsClockConfiguration) clockConfiguration).setWarmUpTime(((JLoadProfileRps) jLoadProfile).getWarmUpTimeInSeconds());
             ((RpsClockConfiguration) clockConfiguration).setMaxThreadNumber((int) ((JLoadProfileRps) jLoadProfile).getMaxLoadThreads());
+        } else if (jLoadProfile instanceof JLoadProfileUserGroups) {
```

Comment:
yes, this PR must be merged after #693.
due to 'aux' I wasn't able to make any changes in code, that's why I had to push that changes also.
don't pay attention to it.